nncf/common/graph/transformations/commands.py

```diff
@@ -63,6 +63,8 @@ class TargetType(IntEnum):
                           specific module attributes - N/A in TF
     `OPERATOR_POST_HOOK` - a location after a function call in PT without access to
                            specific module attributes - N/A in TF
+    `POST_BRANCH_WITH_PARTIAL_MERGE` - a location after the associated model layer,
+                                       that represents the connection with the specific branches
```

Comment:
By dentition, TargetType describes **the types of locations in the model that can be modified** using NNCF in order to create a compressed model. For example, Torch will insert the same FQ before branch_node_names and ONNX and OpenVINO I hope will do the same.

In my opinion, you should use special command or extension of existing for this that was implemented via OPERATOR_PRE_HOOK in Torch backend.

Reply:
@vshampor, could you please comment on the possibility of the proposal for the Torch backend?

Reply:
It does not seem possible to me that torch backend would be able produce the kind of topology that we need here (with the possibilities of graph modification in torch that we have now).

Reply:
@vshampor can we discuss this offline?